### PR TITLE
feat: dashboard page

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: apps/asap-server/src/controllers/dashboard.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { GraphQL } from '@asap-hub/services-common';
+import { DashboardResponse } from '@asap-hub/model';
+import {
+  CMSNewsAndEvents,
+  parse as parseNewsAndEvents,
+} from '../entities/news-and-events';
+import { CMSPage, parse as parsePage } from '../entities/page';
+
+export const query = `
```
Comment (jeysal) [inline apps/asap-server/src/controllers/dashboard.ts:9]: I'm wondering:
If we can query Squidex via GraphQL (even if we end up exposing it as REST from the backend to save implementation time), why did we need to store any redundant data?

File: apps/frontend/src/home/__tests__/Home.test.tsx [hunk 2]
```diff
@@ -23,6 +23,6 @@ const renderHome = async () => {
 };
 
 test('renders asap link', async () => {
-  const { getByRole } = await renderHome();
-  expect(getByRole('heading').textContent).toEqual('Dashboard');
+  const { getByText } = await renderHome();
+  expect(getByText(/welcome to the hub/i, { selector: 'h1' })).toBeVisible();
```
Comment (jeysal) [inline apps/frontend/src/home/__tests__/Home.test.tsx:27]:
```suggestion
  expect(getByText(/welcome/i, { selector: 'h1' })).toBeVisible();
```

File: apps/storybook/src/WhereToStart.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { WhereToStartSection } from '@asap-hub/react-components';
+
+export default {
+  title: 'Organisms / Dashboard / Where To Start',
+};
+
+export const Normal = () => <WhereToStartSection pages={[]} />;
```
Comment (jeysal) [inline apps/storybook/src/WhereToStart.stories.tsx:8]: This is unfortunately useless for the preview, can you add a configurable number of pages here? (using something like `Array.from({length: number('Number of Pages', 2, {min: 0})}, () => {...})`, we have this in places)

File: packages/react-components/src/organisms/__tests__/PageCard.test copy.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React, { ComponentProps } from 'react';
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/PageCard.test copy.tsx:1]: odd file name :sweat_smile:

File: packages/react-components/src/templates/DashboardPageHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  firstName,
+}) => {
+  return (
+    <header css={containerStyles}>
+      <Display
+        styleAsHeading={2}
+      >{`Welcome to the Hub, ${firstName}!`}</Display>
+      <div css={textStyles}>
+        <Paragraph accent="lead">
+          The Hub is the beating hearth of the ASAP network, where all grantees
```
Comment (jeysal) [inline packages/react-components/src/templates/DashboardPageHeader.tsx:37]:
```suggestion
          The Hub is the beating heart of the ASAP network, where all grantees
```

**Review by jeysal — CHANGES_REQUESTED** [review]
CI not passing yet, but had some time for first few comments

File: packages/react-components/src/organisms/WhereToStartSection.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  },
+});
+
+const WhereToStart: React.FC<WhereToStartProps> = ({ pages }) => {
+  return (
+    <section>
+      <Display styleAsHeading={2}>Not sure where to start?</Display>
+      <div css={gridContainerStyles}>
+        {pages.map((page) => (
+          <PageCard {...page} href="#" />
```
Comment (jeysal) [inline packages/react-components/src/organisms/WhereToStartSection.tsx:30]: Would be unfortunate to keep opening more gaps in the product now - do we have no clue where this is supposed to link?

File: packages/react-components/src/templates/DashboardPageBody.tsx [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+    ) : null}
+    <section>
+      <Display styleAsHeading={2}>You may want to try</Display>
+      <div css={containerStyles}>
+        <Card>
+          <ul>
+            <li>
+              <Paragraph primary>
+                Check out grantee profiles on{' '}
+                <Link href="/network/users">Network</Link>
```
Comment (jeysal) [inline packages/react-components/src/templates/DashboardPageBody.tsx:49]: These need to be passed in, component lib does not know about route structure

File: packages/react-components/src/templates/DashboardPageBody.tsx [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+              <Paragraph primary>
+                Rejoice on your fabulous <Link href={hrefProfile}>Profile</Link>
+              </Paragraph>
+            </li>
+          </ul>
+        </Card>
+      </div>
+    </section>
+    <section css={bottomStyles}>
+      <CtaCard href={'#'} buttonText="Contact Us">
```
Comment (jeysal) [inline packages/react-components/src/templates/DashboardPageBody.tsx:89]: more here :/

**Review by jeysal — CHANGES_REQUESTED** [review]
Mostly LGTM now, just some gaps and missing some tests for patch coverage :)

File: packages/react-components/src/templates/DashboardPageBody.tsx (hunk 8 above)
Comment (fampinheiro) [reply packages/react-components/src/templates/DashboardPageBody.tsx:89]: Tracked on trello :slightly_smiling_face:

File: packages/react-components/src/organisms/WhereToStartSection.tsx (hunk 6 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/WhereToStartSection.tsx:30]: The link comes from the API. Not implemented yet, follow up pull request.

**Pull request merged**